provider/README.md

```diff
@@ -336,8 +336,51 @@ When multiple query parameters are specified, they should all apply to the retur
 [Top][toc]
 
 ## Realtime Data
+All providers should provide an endpoint that displays the current status of a providers vehicles on the PROW. Only vehicles that are currently in an available, unavailable, or reserved states should be returned in this payload. Data in this endpoint should generally reconcile with data from the status changes enpdoint.
```

Comment:
Total vehicles in PRoW consists of those in states of available, unavailable, reserved, and trip.  Can vehicles in trip be added to this endpoint to ensure accurate counting for the purposes of things like vehicle caps?

Reply:
For provider reserved covers reserved & trip doesn't it?